proposals/NNNN-root-signature-dx-containe-dx-container-codegen.md

```diff
@@ -0,0 +1,130 @@
+<!-- {% raw %} -->
+
+# Implementation of Root Signatures in DX Container
+
+* Proposal: [NNNN](NNNN-filename.md)
+* Author(s): [Xiang Li](https//github.com/python3kgae), [Damyan
+  Pepper](https://github.com/damyanp), [Joao Saffran](https://github.com/joaosaffran)
+* Status: **Design In Progress**
+
+<!--
+*During the review process, add the following fields as needed:*
+
+* PRs: [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/pull/NNNN)
+* Issues:
+  [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/issues/NNNN)
+  -->
+
+## Introduction
+
+[Purposal 0002](https://github.com/joaosaffran/wg-hlsl/blob/purposal/root-signatures/proposals/0002-root-signature-in-clang.md?plain=1)
```

Comment:
```suggestion
[Proposal 0002](0002-root-signature-in-clang.md)
```

"Proposal" is spelled incorrectly, but more importantly this links to a version in your fork which isn't the right thing to do.